tests/integration-tests/s3_common_utils.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import boto3
+from assertpy import assert_that
+
+
+def _check_s3_read_resource(region, cluster, s3_arn):
+    _check_role_inline_policy(region, cluster, "S3Read", s3_arn)
+
+
+def _check_s3_read_write_resource(region, cluster, s3_arn):
```

Comment:
It looks like `_check_s3_read_resource` and `_check_s3_read_write_resource` are imported by other modules. The convention is to avoid naming interfaces intended to be used by code outside the file with a leading underscore. Can you remove the `_` from the function names and change the imports and usages in test_update.py and test_iam.py?

Reply:
yes, done!